package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "mini-lightbox",
-  "version": "1.3.1",
+  "version": "1.3.2",
   "description": "Minimalist image lightbox",
-  "main": "index.js",
+  "main": "lib/mini-lightbox.js",
```

Comment:
> But that also means everyone not using CommonJS has to use the minified file.

Let's wait for that case. :joy: I hope there are simpler solutions.

> If you can live with that, I'll revert some of the changes accordingly.

Just make new commits to remove these snippets. It should be fine.